Frends.SFTP.WriteFile/Frends.SFTP.WriteFile/Frends.SFTP.WriteFile.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System.ComponentModel;
+using Renci.SshNet;
+
+#pragma warning disable 1591
+
+namespace Frends.SFTP.WriteFile
+{
+    public class SFTP
+    {
+
+        /// <summary>
+        /// Writes a file with SFTP connection.
+        /// Documentation: https://github.com/FrendsPlatform/Frends.SFTP.WriteFile
+        /// </summary>
+        /// <param name="connection">Transfer connection parameters</param>
+        /// <param name="source">Source file location</param>
+        /// <param name="destination">Destination directory location</param>
+        /// <param name="cancellationToken"></param>
+        /// <returns>Result object </returns>
+        public static Result WriteFile([PropertyTab] Source source, [PropertyTab] Destination destination, [PropertyTab] Connection connection, CancellationToken cancellationToken)
+        {
+            return new SFTP().WriteFileInternal(connection, source, destination, cancellationToken);
+        }
+
+        internal Result WriteFileInternal(Connection connection, Source source, Destination destination, CancellationToken cancellationToken)
```

Comment:
Is there some reason why this method is `internal Result WriteFileInternal` and not `private static Result WriteFileInternal`?

Reply:
No idea, fixed